### PR TITLE
#999: Use `scalar.Sticky` for caching request

### PR DESCRIPTION
**Pull request by andreoss**
Per #999:

- Make `RqOnce` use `Sticky`

### CODE REVIEW

**Comment by fabriciofx** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/takes/issues/1076#issuecomment-694106999))

@fabriciofx The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#1076](https://github.com/yegor256/takes/issues/1076), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/takes/issues/1076#issuecomment-694106999))

@fabriciofx Job refused in 9 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

File: src/main/java/org/takes/rq/RqOnce.java [hunk 1]
```diff
@@ -33,9 +37,9 @@
  * <p>The class is immutable and thread-safe.
  *
  * @since 0.36
- * @todo #918:30min Please use {@link org.cactoos.scalar.Sticky} decorator
- *  class so that multiple calls to body() would produce the same cached result
- *  rather than throwing an exception.
+ * @todo #999:30min Please use {@link org.cactoos.text.Sticky} decorator
+ *  class instead of inner anonymous class below, which prevents input
```
Comment (paulodamaso) [inline src/main/java/org/takes/rq/RqOnce.java:41]: @andreoss This seems to be fairly simple to do, couldn't you do this in this same PR?

**Review by paulodamaso — CHANGES_REQUESTED** [review]
@andreoss Just one comment here, please take a look

File: src/main/java/org/takes/rq/RqOnce.java (hunk 1 above)
Comment (andreoss) [reply src/main/java/org/takes/rq/RqOnce.java:41]: @paulodamaso No, this class is not yet released.

**Comment by andreoss** [comment]
@paulodamaso Please take a look one more time.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/takes#1076` is not assigned, can't get performer

**Pull request merged**